### PR TITLE
chore(Linter): Update ESLint to version 3 and configuration

### PR DESCRIPTION
**Pull request by baldore**
Updates ESLint package. Change ESLint configuration base to Airbnb in order to apply ES2015 style.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "airbnb",
+  "plugins": [
+    "react"
```
Comment (andresdavid90) [inline .eslintrc.json]: Do we need this?

File: package.json [hunk 2]
```diff
@@ -36,9 +36,11 @@
     "codecov.io": "0.1.6",
     "commitizen": "2.7.4",
     "cz-conventional-changelog": "1.1.5",
-    "eslint": "1.10.3",
-    "eslint-config-defaults": "^9.0.0",
-    "eslint-plugin-filenames": "^0.2.0",
+    "eslint": "3.0.1",
+    "eslint-config-airbnb": "9.0.1",
+    "eslint-plugin-import": "1.10.2",
+    "eslint-plugin-jsx-a11y": "1.5.5",
```
Comment (andresdavid90) [inline package.json]: And this? 🙄

File: src/data-adapter.js [hunk 3]
```diff
@@ -14,12 +12,10 @@ const parseDirPath = (dirPath) => {
...
-      path: url.parse(link).path,
-      fileName: `${parseDirPath(savePath)}${path.basename(link)}`
-    };
-  });
+module.exports = function getDataAdapterOptions(data, savePath = process.cwd()) {
+  return Array.from(new Set(data)).map((link) => ({
+    host: url.parse(link).host,
+    path: url.parse(link).path,
+    fileName: `${parseDirPath(savePath)}${path.basename(link)}`
+  }));
```
Comment (andresdavid90) [inline src/data-adapter.js]: ✌🏻

File: .eslintrc.json (hunk 1 above)
Comment (baldore) [inline .eslintrc.json]: Those are dependencies of the airbnb styleguilde.

File: package.json (hunk 2 above)
Comment (baldore) [inline package.json]: Those plugins are used by the Airbnb styleguide and required in the project.

File: .eslintrc.json (hunk 1 above)
Comment (andresdavid90) [inline .eslintrc.json]: Oh ok.

File: package.json (hunk 2 above)
Comment (andresdavid90) [inline package.json]: 💂🏻

**Pull request merged**